tests/components/hassio/test_init.py

```diff
@@ -715,21 +721,27 @@ async def test_coordinator_updates(hass, caplog):
 
 
 @pytest.mark.parametrize(
-    "os_info",
+    "extra_os_info, integration",
     [
-        {
-            "json": {
-                "result": "ok",
-                "data": {"version_latest": "1.0.0", "version": "1.0.0", "board": "rpi"},
-            }
-        }
+        ({"board": "odroid-c2"}, "hardkernel"),
+        ({"board": "odroid-c4"}, "hardkernel"),
+        ({"board": "odroid-n2"}, "hardkernel"),
+        ({"board": "odroid-xu4"}, "hardkernel"),
+        ({"board": "rpi"}, "raspberry_pi"),
+        ({"board": "rpi0"}, "raspberry_pi"),
+        ({"board": "rpi0-w"}, "raspberry_pi"),
+        ({"board": "rpi2"}, "raspberry_pi"),
```

Comment:
```suggestion
```

Reply:
I'd leave the `rpi2` in for now as we still build OS images for it. It's not officially supported anymore, but we have a sufficient large user base (439 according to analytics) and it doesn't cost me a lot of effort to keep supporting it right now.

Reply:
OK, `rpi2` is added back now.